deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java

```diff
@@ -161,7 +137,6 @@ public boolean test(BeanInfo beanInfo) {
 
         @Override
         public void capture(String name, byte[] bytes) {
-            ClassWriter file = new ClassWriter(ClassWriter.COMPUTE_FRAMES | ClassWriter.COMPUTE_MAXS);
```

Comment:
I'm not sure if this had any side-effect but the variable was completely unused and the tests still pass, so ....

Reply:
this can been removed